Library/Homebrew/cmd/update-report.rb

```diff
@@ -118,7 +118,7 @@ def update_report
       if hub.empty?
         puts "No changes to formulae."
       else
-        hub.dump
+        hub.dump unless args.preinstall?
```

Comment:
I think this is still worth outputting just without the modified formulae? The added/deleted/renamed ones would still be of interest. Could note if needed that we're not outputting modified ones.